### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by tiagojsag**
Homepage + Styling

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,15 +6,33 @@
 
 # Ignore bundler config.
 /.bundle
+/.gems
+/.rbenv-gemsets
+
+# Ignore MacOSX files
```
Comment (tiagojsag) [inline .gitignore:12]: OS/IDE specific files should be ignored globally, not here.

`echo ".DS\_Store" >> ~/.gitignore && git config --global core.excludesfile ~/.gitignore`

**Comment by tiagojsag** [comment]
I already removed the DS_Store files, but please be careful with those in the future

File: config/initializers/assets.rb [hunk 2]
```diff
@@ -9,3 +9,6 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
+
+# Set fonts to precompile
+Rails.application.config.assets.precompile += %w( TradeGothicLTCom-Bold.ttf )
```
Comment (tiagojsag) [inline config/initializers/assets.rb]: missing some fonts here?

File: db/schema.rb [hunk 3]
```diff
@@ -1,4 +1,3 @@
-# encoding: UTF-8
```
Comment (tiagojsag) [inline db/schema.rb:1]: ??

File: package.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "onboarding",
```
Comment (tiagojsag) [inline package.json:2]: update README with `npm install` + instructions to generate assets for prod env.

File: .editorconfig [hunk 5]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (tiagojsag) [inline .editorconfig:1]: next time, please focus on feature branches. `.editorconfig` is totally fine, just dont add it as part of something else

File: db/schema.rb (hunk 3 above)
Comment (simaob) [inline db/schema.rb:1]: I'm guessing this was done by the computer =D

**Comment by simaob** [comment]
hey guys, have all comments been addressed? Can we re-review?

**Comment by davidsingal** [comment]
@bitTal is fixing some precompiling errors, when is done she will update this PR

**Comment by simaob** [comment]
Thanks! kudos

**Comment by davidsingal** [comment]
For me, now it is ready to merge. Other improvements coming soon.

**Pull request merged**